src/crate/crash/command.py

```diff
@@ -306,7 +273,10 @@ def _connect(self, server):
             self.logger.critical('CONNECT ERROR')
         else:
             self.logger.info('CONNECT OK')
-            self._check()
+
+            # check for failing checks in a cluster
+            check = built_in_commands.get("check")
```

Comment:
you can access the item directly using `built_in_commands['check']` because `built_in_commands` is not dynamically changed